dtests/src/test/java/io/snappydata/hydra/testDMLOps/SnappyDMLOpsUtil.java

```diff
@@ -0,0 +1,1049 @@
+      String dataLocation = SnappySchemaPrms.getDataLocations();
+      for (int i = 0; i < tableNames.length; i++) {
+        String tableName = tableNames[i].toUpperCase();
+        Log.getLogWriter().info("Loading data into " + tableName);
+        String csvFilePath = dataLocation + File.separator + csvFileNames[i];
+        Log.getLogWriter().info("CSV location is : " + csvFilePath);
+        FileInputStream fs = new FileInputStream(csvFilePath);
+        BufferedReader br = new BufferedReader(new InputStreamReader(fs));
+        String insertStmt = "insert into " + tableName + " values (";
+        String row = null;
+
+        ArrayList<Integer> dmlthreads = null;
+        int tid;
+        if (SnapshotIsolationDMLOpsBB.getBB().getSharedMap().containsKey("dmlThreads"))
+          dmlthreads = (ArrayList<Integer>)SnapshotIsolationDMLOpsBB.getBB().getSharedMap().get("dmlThreads");
+        while ((row = br.readLine()) != null) {
+          if (dmlthreads == null)
+            tid = getMyTid();
+          else
+            tid = dmlthreads.get(new Random().nextInt(dmlthreads.size()));
+          String rowStmt = insertStmt + row + "," + tid + ")";
+          //Log.getLogWriter().info("Row is : " +  rowStmt);
+          conn.createStatement().execute(rowStmt);
+          dConn.createStatement().execute(rowStmt);
+        }
+        Log.getLogWriter().info("Done loading data into table " + tableName);
+      }
+      conn.close();
+      derbyTestUtils.closeDiscConnection(dConn, true);
+    } catch (IOException ie) {
+      throw new TestException("Got exception while populating table.", ie);
+    } catch (SQLException se) {
+      throw new TestException("Got exception while populating table.", se);
+    }
+  }
+
+  protected void dropTables(Connection conn) {
+    String sql = null;
+    String[] tables = SnappySchemaPrms.getTableNames();
+    sql = "drop table if exists ";
+    try {
+      for (String table : tables) {
+        Statement s = conn.createStatement();
+        s.execute(sql + table);
+      }
+    } catch (SQLException se) {
+      throw new TestException("Got exception while dropping table.", se);
+    }
+  }
+
+  /*
+   Hydra task to perform DMLOps which can be insert, update, delete
+   */
+  public static void HydraTask_performDMLOp() {
+    testInstance.performDMLOp(ConnType.JDBC);
+  }
+
+  public void performDMLOp(ConnType connType) {
+    //perform DML operation which can be insert, update, delete.
+    String operation = SnappySchemaPrms.getDMLOperations();
+    switch (SnappyDMLOpsUtil.DMLOp.getOperation(operation)) {
+      case INSERT:
+        Log.getLogWriter().info("Performing insert operation.");
+        performInsert();
+        break;
+      case UPDATE:
+        Log.getLogWriter().info("Performing update operation.");
+        if (connType.equals(ConnType.JDBC))
+          performUpdate();
+        else
+          performUpdateInSnappy(connType);
```

Comment:
Rename method names. Seems quite confusing.  e.g. performUpdate() to performUpdateUsing JDBC()
and performUpdateInSnappy() to performUpdateUsingSNC()